CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 2.26.0 (May 5th, 2025)
+
+[Full Changelog](https://github.com/confluentinc/terraform-provider-confluent/compare/v2.25.0...v2.26.0)
+
+**New features:**
+* Added `disable_wait_for_ready` attribute for `confluent_role_binding` [resource](https://registry.terraform.io/providers/confluentinc/confluent/latest/docs/resources/confluent_role_binding) and [data-source](https://registry.terraform.io/providers/confluentinc/confluent/latest/docs/data-sources/confluent_role_binding) to disable the readiness check.
```

Comment:
```suggestion
* Added `disable_wait_for_ready` attribute for `confluent_role_binding` [resource](https://registry.terraform.io/providers/confluentinc/confluent/latest/docs/resources/confluent_role_binding) to disable the readiness check.
```

This should only be for the resource since this attribute is related to provisioning.